homeassistant/components/totalconnect/binary_sensor.py

```diff
@@ -101,6 +101,21 @@ class TotalConnectAlarmBinarySensorEntityDescription(BinarySensorEntityDescripti
         entity_category=EntityCategory.DIAGNOSTIC,
         is_on_fn=lambda location: location.is_ac_loss(),
     ),
+    TotalConnectAlarmBinarySensorEntityDescription(
+        key="smoke",
+        device_class=BinarySensorDeviceClass.SMOKE,
+        is_on_fn=lambda location: location.arming_state.is_triggered_fire(),
+    ),
+    TotalConnectAlarmBinarySensorEntityDescription(
+        key="carbon_monoxide",
+        device_class=BinarySensorDeviceClass.CO,
+        is_on_fn=lambda location: location.arming_state.is_triggered_gas(),
+    ),
+    TotalConnectAlarmBinarySensorEntityDescription(
+        key="police",
+        device_class=BinarySensorDeviceClass.SAFETY,
```

Comment:
Not sure this make sense. Is it unsafe that police is present? Add a better translation key and remove that class.

Reply:
On the physical alarm panel, there is a button that says Police. When pushed, if you have an alarm monitoring service, the police would be called.  Monitoring or not, the library's `is_triggered_police()` is True.

This binary sensor would be On/True once the button is pressed.  It stays on until the alarm is "cleared".

"Unsafe" seemed like the best option from BinarySensorDeviceClass.  I would prefer to use Police as the key because it would show up in HA with a name matching the physical hardware.

Maybe change it to "Police Button" to be more obvious?

Reply:
Just dont use any device class. Set your own translations that match what it is.